assets/js/src/newsletters/templates.jsx

```diff
@@ -35,10 +35,6 @@ if (window.mailpoet_woocommerce_active) {
 
 templatesCategories.push(
   ...[
-    {
-      name: 'all',
-      label: MailPoet.I18n.t('allTemplates'),
```

Comment:
We should also remove `allTemplates` from translations on `newsletters.html` view.